File: PHP/PHP Assignment -02/A10.php
<!-- Write a php program to find the largest no 
among three numbers.  -->
<?php

echo $a=100 . "<br>";
echo $b=200 . "<br>";
echo $c=180 . "<br>";

if($a>$b&&$a>$c)
{
    echo "Largest Number is : ". $a;
}
else if($b>$a&&$b>$c)
{
    echo "Largest Number is : ". $b;
}
else
{

    echo "Largest Number is : ". $c;
}

?>

File: C#/a10.cs
//10. WCP to find the largest number among three numbers......................................
    using System;
    namespace a10;
    class Program
    {
        static void Main()
        {
            Console.WriteLine("Enter three numbers:");
            int num1 = Convert.ToInt32(Console.ReadLine());
            int num2 = Convert.ToInt32(Console.ReadLine());
            int num3 = Convert.ToInt32(Console.ReadLine());

            int largest = FindLargest(num1, num2, num3);
            Console.WriteLine("The largest number is: " + largest);
        }
        static int FindLargest(int a, int b, int c)
        {
            if (a >= b && a >= c)
                return a;
            else if (b >= a && b >= c)
                return b;
            else
                return c;
        }
    }